trade-finance-manager-ui/server/routes/utilisation-reports/index.ts

```diff
@@ -33,13 +33,7 @@ utilisationReportsRoutes.post(
   updateUtilisationReportStatus,
 );
 
-utilisationReportsRoutes.get(
-  '/:id/download',
-  validateTfmPaymentReconciliationFeatureFlagIsNotEnabled,
-  validateUserTeam(Object.values(PDC_TEAM_IDS)),
-  validateSqlId('id'),
-  getReportDownload,
-);
+utilisationReportsRoutes.get('/:id/download', validateUserTeam(Object.values(PDC_TEAM_IDS)), validateSqlId('id'), getReportDownload);
```

Comment:
The `validateTfmPaymentReconciliationFeatureFlagIsNotEnabled` check has been removed, is this intentional please?

Reply:
Yes, we used to be able to download on a previous page, but when we made the changes to that page that are behind the feature flag we removed the link and so we put the route behind this check for security, however that was when we didn't know there would be a requirement to allow downloads after turning on the feature flag, just on a different page. 

So it is correct to allow it with it with the flag off, because the landing page has downloads in that case, and with the flag on, as this page will have downloads in that case.

Reply:
OK great thanks @RLavender98